scripts/corelli/calibration/database.py

```diff
@@ -7,118 +7,198 @@
     """
-    # verify timestamps are correct
-    verify_date_YYYYMMDD('save_manifest_file', date)
-    filename = database_path + '/manifest_corelli_' + date + '.csv'
+    # verify day_stamps are correct
+    [verify_date_format('save_manifest_file', date) for date in day_stamps]
+    if manifest_day_stamp is None:
+        manifest_day_stamp = str(sorted(day_stamps)[-1])
+    verify_date_format('save_manifest_file', manifest_day_stamp)
+
+    filename = database_path + '/manifest_corelli_' + str(manifest_day_stamp) + '.csv'
 
-    if pathlib.Path(filename).is_file():
-        file = open(filename, 'a+')
-    else:
-        file = open(filename, 'w')
-        file.write('bankID, timestamp\n') # header
+    file = open(filename, 'w')
+    file.write('bankID, timestamp\n')  # header
 
-    lines:str = ''
-    for t, bankID in enumerate(bankIDs):
-        lines += str(bankID) + ', ' + str(date) + '\n'
+    lines: str = ''
+    for bank_id, date in zip(bank_ids, day_stamps):
+        lines += str(bank_id) + ', ' + str(date) + '\n'
 
     file.write(lines)
     file.close()
+    return filename
 
 
-def save_bank_table( data:Workspace, bankID:int, database_path:str, date:str,
-                     table_type:str = 'calibration') -> None:
+def save_bank_table(data: Workspace, bank_id: int, database_path: str, date: str,
+                    table_type: str = 'calibration') -> None:
     """
     Function that saves a bank calibrated TableWorkspace into a single HDF5 file
     using corelli format and using current date:
     database_path/bank0ID/type_corelli_bank0ID_YYYYMMDD.nxs.h5
     :param data input Workspace (TableWorkspace) data for calibrated pixels
-    :param bankID bank number that is calibrated
+    :param bank_id bank number that is calibrated
+    :param date format YYYYMMDD
     :param database_path location of the corelli database (absolute or relative)
            Example: database/corelli/ for
                     database/corelli/bank001/
                     database/corelli/bank002/
     :param table_type 'calibration', 'mask' or 'fit'
     """
-    verify_date_YYYYMMDD('save_bank_table', date)
-    filename:str = filename_bank_table(bankID, database_path, date, table_type)
-    SaveNexusProcessed(data,filename)
-
-
-def load_bank_table( bankID:int, database_path:str, date:str, table_type:str = 'calibration'
-                     ) -> TableWorkspace:
+    verify_date_format('save_bank_table', date)
+    filename: str = filename_bank_table(bank_id, database_path, date, table_type)
+    SaveNexusProcessed(data, filename)
+
+
+def save_calibration_set(input_workspace: InputWorkspaceTypes,
+                         database_path: str,
+                         calibrations: CalibrationInputSetTypes,
+                         masks: Optional[CalibrationInputSetTypes] = None,
+                         fits: Optional[CalibrationInputSetTypes] = None) ->None:
+    r"""
+    Save one or more calibration workspaces to the database
```

Comment:
Please describe the expected output in the database including the path from the database base directory. Something like `database/corelli/bank001/file1 file2 file3 `, see the docs I put for save/load functions. I assume this is for the acceptance criteria, so the user know what to expect.

Reply:
I completely forgot to write a decent docstring. I just added it